packages/react-app/src/types/Filter.ts

```diff
@@ -6,17 +6,21 @@ export type AcceptedSortInputs = 'reward' | 'createdAt';
 export type AcceptedSortOutputs = `reward.${StringKeysBountyBoardReward}` | '_id' | 'createdAt';
 export interface FilterParams {
 	customerId?: string;
-	status?: string;
+	status?: string
 	search: string;
-	$lte?: number;
-	$gte?: number;
+	lte?: number;
+	gte?: number;
 	sortBy?: string;
-	asc: boolean;
+	asc: boolean | string;
```

Comment:
I thought the creation/use of `booleanAsc` method was spot on, which contradicts adding `string` as a supported type for `asc`